cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt

```diff
@@ -67,7 +67,7 @@ internal class ConsentWebView(
     private fun setup() {
         when {
             viewId != null -> { id = viewId }
-            Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1 -> { id = View.generateViewId() }
+            viewId == null -> { id = View.generateViewId() }
```

Comment:
Just for an extra layer of security, test manually if this point is reached when the id is set from the client app, currently we don’t have an integration test for it

Reply:
Ok, one moment, will do